augraphy/augmentations/markup.py

```diff
@@ -150,7 +150,6 @@ def __call__(self, image, layer=None, force=False):
                     # # dividing the line into points
                     points_count = random.randint(3, 10)  # dividing the line into points
                     points = np.linspace(starting_point[0], ending_point[0], points_count)
-                    relative_start_point = starting_point[1] if i % 2 else ending_point[1]
```

Comment:
@shaheryar1 This line should be not needed right? I don't see any usage of `relative_start_point`.